apps/BosGenaDrop/widget/Mintbase/App/Activity/Analytics.jsx

```diff
@@ -0,0 +1,308 @@
+    }
+  }
+`;
+
+const Row = styled.div`
+    margin-top:20px;
+    display:flex;
+    flex-direction: row; 
+    gap:20px;
+    flex-wrap: nowrap;
+    width:100%;
+    justify-content: center;
+`;
+
+const ContainerCard = styled.div`
+  .info-card {
+    border-radius: 0.25rem;
+    background: ${IsDarkModeOn ? "var(--gray-800)" : "rgba(40,42,58,0.1)"};
+    //max-width: 165px;
+    min-width: 168px;
+    
+    &.small,
+    &.medium,
+    &.big {
+      padding: 8px;
+      min-height: 46px;
+      width:100%;
+      @media (min-width: 480px) {
+        min-height: 72px;
+        padding: 12px;
+      }
+    }
+    @media (min-width: 976px) {
+      .info-card.small {
+        min-width: 160px;
+      }
+    }
+    .title-wrapper {
+      display: flex;
+      position: relative;
+      align-items: center;
+    }
+    
+
+`;
+
+const Title = styled.div`
+  color: ${IsDarkModeOn ? "var(--gray-300)" : "var(--gray-700)"};
+  font-size: 13px;
+`;
+const Description = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: baseline;
+  width: 100%;
+  magrin: 5px 0;
+  .description-rt {
+    display: flex;
+    width: 100%;
+    font-size:18px;
+    font-weight: 600;
+  }
+`;
+
+
+const SearchBar = styled.div`
+    padding: 20px;
+    margin-top: 20px;
+    diplay:flex;
+    flex-direction:column;
+    gap:1rem;
+    background: #dcdcdc;
+    .title{
+        font-size:18px;
+    }
+    .search{
+        width:100%;
+        padding: 10px 20px;
+        outline:none;
+        border: none;
+        background:rgba(243, 244, 248);
+        padding: 12px;
+        margin:10px 0;
+        border-radius:5px;
+        :focs{
+            border: 1px soild gray;
+            
+        }
+    }
+`;
+
+
+const MbCard = ({title,description})=>{
+    return(
+        title&&description&&
+        <ContainerCard>
+            <div className="info-card medium">
+                <div className={`title-wrapper`}>
+                    <Title>{title}</Title>
+                </div>
+                <Description>
+                    <div className="description-rt">{description}</div>
+                </Description>
+            </div>     
+        </ContainerCard>
+    )
+}
+
+const data = [
+    {
+        title: "Mintbase Contracts",
+        description: "4,644,3773"
+    },
+    {
+        title: "Total Contracts",
+        description: "4,644,3773"
+    },
+    {
+        title: "Total Volume",
+        description: "4,644,3773"
+    },
+    {
+        title: "Tokens",
+        description: "4,644,3773"
+    },
+    {
+        title: "Minters",
+        description: "4,644,3773"
+    },
+    {
+        title: "Listed",
+        description: "4,644,3773"
+    },
+    {
+        title: "Affiliate",
+        description: "4,644,3773"
+    },
+]
+
+
+return(
+    <>
+        <SearchBar>
+            <div className="title">Months</div>
+            <input type="search" className="search"  placeholder="0"/>
```

Comment:
is the searchbar integrated ? because it needs to be

Reply:
It means you need to connect that page to that Navbar, sho the Navbar shows when you view it @louisdevzz

Reply:
yes, thanks for answer